webpack/assets/javascripts/react_app/components/HostDetails/HostDataProptypes.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+
+export const hostDataProptypes = {
```

Comment:
Why have you decided to go with a separate file for props definition? Isn't it confusing?
Or is this a new best practice?

Reply:
The separation of the host data Proptypes is for keeping the code dry (it's the same  Proptypes in other host-related components, which can be imported).